rootfs/rootfs/bootscript.sh

```diff
@@ -15,7 +15,10 @@
 
 mkdir -p /var/lib/boot2docker/log
 
-#import settings from profile (or unset them)
+# Add any custom certificate chains for secure private registries
+/etc/rc.d/install_ca_certs
```

Comment:
`tce-loader`, `cgroupfs-mount`, `automount-shares` suggest that this should be `install-ca-certs` :smile: